.github/workflows/testpypi-publish-on-pr.yml

```diff
@@ -27,3 +27,22 @@ jobs:
     - name: Test with pytest
       run: |
         CONF_FILE=tests/unit/conf.ini pytest --log-cli-level=INFO
+    - name: Install pypa/build
+      run: >-
+        python -m
+        pip install
+        build
+        --user
+    - name: Build a binary wheel and a source tarball
+      run: >-
+        python -m
+        build
+        --sdist
+        --wheel
+        --outdir dist/
+        .
+    - name: Publish distribution 📦 to Test PyPI
```

Comment:
it seems artifact version pushed to testpy is standartd (doesn't contain PR number [1]), this might yield much mess if there's multiple PR's at same time?

[1] https://test.pypi.org/project/kensu/#history